src/service/resources-impl.js

```diff
@@ -859,6 +871,19 @@ export class Resources {
     this.visibilityStateMachine_.setState(
       this.viewer_.getVisibilityState()
     );
+
+    if (this.maybeChangeHeight_) {
+      this.maybeChangeHeight_ = false;
+      this.vsync_.measure(() => {
```

Comment:
Nit: We can still queue up multiple measures here. We can add a second boolean `hasMeasuredDocHeight`, setting to false when we queue, and setting to true when it's measured. Then, when we see that `maybeChangeHeight_ && !hasMeasuredDocHeight`, we know we can skip since it's already queued.